integration-tests/test_file_workflow.py

```diff
@@ -4,6 +4,9 @@
 from constants import HOST_DETAILS
 from constants import MACHINE_ID_FILE
 import conftest
+import logging
```

Comment:
we can replace it with pytest_client_tools logged_run

Reply:
I will replace it in my next PR where I will rewrite usage of --check-results command to get inventory data.